CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 This document lists new features, improvements, changes, and bug fixes in each release of the package.
 
+## GDScript mode 1.?.?
```

Comment:
1.1.1 isn't out so we can replace the section below this one in the changelog with 1.2.0 and merge these together.